### PR TITLE
Permission update delayed - MTA 7.0.0 known issues

### PR DESCRIPTION
**Pull request by HagayVider1**
OADP-1809
Replaces https://github.com/windup/windup-documentation/pull/791

### CODE REVIEW

File: docs/topics/mta-rn-known-issues-7-0-0.adoc [hunk 1]
```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: REFERENCE
+[id="rn-known-issues-7-0-0_{context}"]
+= Known issues
+
+MTA version 7.0.0 has the following issues.
```
Comment (anarnold97) [inline docs/topics/mta-rn-known-issues-7-0-0.adoc:9]:
```suggestion
MTA version 7.0.0 has the following known issues.
```

File: docs/topics/mta-rn-known-issues-7-0-0.adoc [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+In the `Migration -> Dependencies` view, some rows show a placeholder under the `version` column instead of the actual value. link:https://issues.redhat.com/browse/MTA-1283[MTA-1283]
+
+.Source analysis shows different issues for `tackle-testapp`
+
+Running a source analysis using `tackle-testapp` with target as `containerization` or `cloud-readiness` shows missing categories. `Issues` has only `Mandatory` category with two issues, and no `Information` category. link:https://issues.redhat.com/browse/MTA-1271[MTA-1271]
+
+.Delayed permission update and user deactivation in `RBACK`
+
+When deleting, deactivating or degarding the role of a user, such as changing the user from `Admin` to `Migrator`, the change takes several minutes to take effect. This delay in changing the user status may pose an operational or security risk. link:https://issues.redhat.com/browse/MTA-1809[MTA-1809]
```
Comment (anarnold97) [inline docs/topics/mta-rn-known-issues-7-0-0.adoc:57]:
```suggestion
When deleting, deactivating or degarding the role of a user, such as changing the user from `Admin` to `Migrator`, the change takes several minutes to take effect. This delay in changing the user status may pose an operational or security risk. link:https://issues.redhat.com/browse/MTA-1809[MTA-1809]
```

Not sure **degarding** is a word

**Comment by HagayVider1** [comment]
/ label peer-review-needed
See preview here: https://deploy-preview-745--windup-documentation.netlify.app/docs/release-notes/master/#rn-known-issues-7-0-0_release-notes

File: docs/topics/mta-rn-known-issues-7-0-0.adoc [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+It is not possible to remove a Jira connection to a migration wave after that wave is removed. link:https://issues.redhat.com/browse/MTA-1014[MTA-1014]
+
+.Binary analysis takes three hours and then fails
+
+Binary analysis can take three hours, and then it fails. link:https://issues.redhat.com/browse/MTA-1183[MTA-1183]
+
+.Allow architects to review an application without an assessment
+
+The MTA user can decide to skip the Assessment stage of the application migration. link:https://issues.redhat.com/browse/MTA-115[MTA-115]
```
Comment (anarnold97) [inline docs/topics/mta-rn-known-issues-7-0-0.adoc:29]: I am sorry but this is badly written

Address directly

File: docs/topics/mta-rn-known-issues-7-0-0.adoc [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+MTA does not support disconnected installations. link:https://issues.redhat.com/browse/MTA-17[MTA-17]
+
+.Cannot remove the Jira connection to a migration wave that is later removed 
+
+It is not possible to remove a Jira connection to a migration wave after that wave is removed. link:https://issues.redhat.com/browse/MTA-1014[MTA-1014]
+
+.Binary analysis takes three hours and then fails
+
+Binary analysis can take three hours, and then it fails. link:https://issues.redhat.com/browse/MTA-1183[MTA-1183]
```
Comment (anarnold97) [inline docs/topics/mta-rn-known-issues-7-0-0.adoc:25]: - Alejandro Brugarolas- This issue is no longer reproducible on MTA 7.0.0-28

File: docs/topics/mta-rn-known-issues-7-0-0.adoc (hunk 3 above)
Comment (HagayVider1) [reply docs/topics/mta-rn-known-issues-7-0-0.adoc]: changed

File: docs/topics/mta-rn-known-issues-7-0-0.adoc [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+No `Update Notification` appears at top of window after the fields `Application`, `Job Function`, and `Business services` are updated. link:https://issues.redhat.com/browse/MTA-1024[MTA-1024]
+
+.Disabled `rank` field when creating a new tag category
+
+After creating a new tag category, the `rank` field text box is disabled. link:https://issues.redhat.com/browse/MTA-1095[MTA-1095]
+
+.Cannot create a Jira instance behind a proxy
+
+Red Hat Jira instance (issues.stage.redhat.com) is accessible only through a proxy, but general proxy settings do not work with this configuration. link:https://issues.redhat.com/browse/MTA-849[MTA-849]
```
Comment (anarnold97) [inline docs/topics/mta-rn-known-issues-7-0-0.adoc:46]: Is it only Red Hat Jira instances?

File: docs/topics/mta-rn-known-issues-7-0-0.adoc [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+:_content-type: REFERENCE
+[id="rn-known-issues-7-0-0_{context}"]
+= Known issues
+
+MTA version 7.0.0 has the following known issues.
+
+.Lack of unique IDs and Aria-labels on sections and categories hinders automation
+
+There is a lack of unique IDs applied to sections and categories, which hinders migration automation. link:https://issues.redhat.com/browse/MTA-1203[MTA-1203]
```
Comment (HagayVider1) [inline docs/topics/mta-rn-known-issues-7-0-0.adoc:13]: Remove

**Comment by anarnold97** [comment]
@HagayVider1 this PR is closed. Do not add further commits

**Pull request closed without merging**